app/views/fragments/checkout/fieldsPersonal.scala.html

```diff
@@ -37,6 +38,17 @@
     }
 </div>
 
+    @* ===== Phone ===== *@
+    @if(touchpointBackendResolution.validTestUserCredentialOpt){
+<div class="form-field js-checkout-phone-number">
+    <label class="label" for="telephoneNumber">Phone Number</label>
```

Comment:
`for` should reference the id of the input, not the name
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label